README.md

```diff
@@ -108,10 +109,11 @@ The following inputs are recommended or required for this action:
 
 | Input           | Description                                                                                                                                                                                                      | Required |
 | --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
-| `viewerUrl`     | A URL pointing to the "cfg" API endpoint of the TICS Viewer. It contains the name of the TICS Analyzer Configuration or "-" in case of the default configuration.                                                | true     |
-| `mode`          | Set the mode to run the action in. Options are `client` or `qserver` for an analysis run and `diagnostic` for a diagnostic run to test the setup. The default is `client`.                                       | true     |
+| `mode`          | Set `mode` to `qserver` run the action in TICSQServer mode. Options are `client` or `qserver` for an analysis run and `diagnostic` for a diagnostic run to test the setup. The default is `client`.              | true     |
 | `project`       | Name of the TICS project present in the TICS Viewer.                                                                                                                                                             | true     |
-| `ticsAuthToken` | Authentication token to authorize the plugin when it connects to the TICS Viewer. (Only required if a token is needed to run TICS.)                                                                              | false    |
+| `viewerUrl`     | A URL pointing to the "cfg" API endpoint of the TICS Viewer. It contains the name of the TICS Analyzer Configuration or "-" in case of the default configuration.<br><br>Example: <pre lang="yaml">viewerUrl: https://domain.com/tiobeweb/TICS/api/cfg?name=config</pre>     | true     |
+| `branchdir`            | Root directory of the source files for the branch. TICS stores the last used branch directory in the database and uses this last known directory if this property is not set. It is `recommended` to set this variable. <br><br>Example:<pre lang="yaml">branchdir: ${{ GITHUB_WORKSPACE }}</pre>                              | false       |

```

Comment:
Is it recommended to set a path here, or is it better to use .?
As this option is also used by TICSQServer in later runs where this option has not been set.

Reply:
See other counter comment :)